modules/@apostrophecms/i18n/index.js

```diff
@@ -70,7 +70,9 @@ module.exports = {
     i18n: {
       ns: 'apostrophe',
       browser: true
-    }
+    },
+    // If true, slugifying will strip accents from Latin characters
+    stripUrlAccents: false
```

Comment:
This and https://github.com/apostrophecms/apostrophe/pull/5137/files#diff-3520f0c3913f7e40c36f796cec657de7f9667bc7950f042a2490c236a89b9efbR54 will not allow someone to enable/disable this feature per-locale as stated in the ticket.

Reply:
This was a request from Tom, simple mvp, no per language control.